suite-native/module-assets/src/components/AssetsOverview.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { CryptoIcon, IconName } from '@trezor/icons';
+import {
+    Box,
+    Card,
+    Input,
+    InputWrapper,
+    SelectableListItem,
+    Text,
+    VStack,
+} from '@suite-native/atoms';
+import { AccountInfo } from '@trezor/connect';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+
+type AssetsOverviewProps = {
+    accountInfo: AccountInfo;
+    selectedDevice: string;
```

Comment:
Probably some type alias will be better, I am not sure what this is. Is it like DeviceId or something?

Reply:
Fixed: 
`selectedDevice: DummyDevice | undefined;`